server/tools/docker-entrypoint.sh

```diff
@@ -104,6 +104,9 @@ if echo "$@" | grep -E -v -- '-c |-c=|--server-config |--server-config='; then
     SYS_PROPS+=" -c=standalone-ha.xml"
 fi
 
+# Adding support for JAVA_OPTS_APPEND
+sed -i '$a\\n# Append to JAVA_OPTS. Necessary to prevent some values being omitted if JAVA_OPTS is defined directly\nJAVA_OPTS=\"\$JAVA_OPTS \$JAVA_OPTS_APPEND\"' /opt/jboss/keycloak/bin/standalone.conf
```

Comment:
This seems to be very fragile. Perhaps you could add this at the end of the file?

Reply:
Thanks @slaskaw for reviewing. Adding support for this option will allow to add profile features to Keycloak https://www.keycloak.org/docs/latest/server_installation/#profiles. For my particular scenario, I would need to enable upload_script profile.

Reply:
> This seems to be very fragile. Perhaps you could add this at the end of the file?
Yes, I agree. But the standalone.conf file is not part of the repo and is extracted from the Keycloak binary which is downloaded and installed during the build.

Reply:
Ok, got it.